src/pretix/static/pretixpresale/js/widget/widget.js

```diff
@@ -500,7 +501,9 @@ Vue.component('item', {
         // Availability
         + '<div class="pretix-widget-item-availability-col">'
         + '<a v-if="show_toggle" :href="\'#\' + item.id + \'-variants\'" @click.prevent.stop="expand" role="button" tabindex="0"'
-        + '   v-bind:aria-expanded="expanded ? \'true\': \'false\'" v-bind:aria-controls="item.id + \'-variants\'">'+ strings.variations + '</a>'
+        + '   v-bind:aria-expanded="expanded ? \'true\': \'false\'" v-bind:aria-controls="item.id + \'-variants\'">'
+        + '{{ variationsLink }}'
+        + '</a>'
```

Comment:
I would call it variationToggleLabel – Link sounds like a link or URL.

```suggestion
        + '   v-bind:aria-expanded="expanded ? \'true\': \'false\'" v-bind:aria-controls="item.id + \'-variants\'">{{ variationsToggleLabel }}</a>'
```